frontend/public/components/storage-class-form.tsx

```diff
@@ -834,6 +833,10 @@ export class StorageClassForm_ extends React.Component<
     }
 
     const dynamicContent = _.map(parameters, (parameter, key) => {
+      const paramId = `storage-class-provisioner-${_.get(parameter, 'name', key).replace(
```

Comment:
Maybe `_.kebabCase`?

```suggestion
      const paramID = `storage-class-provisioner-${_.kebabCase(_.get(parameter, 'name', key))}`;
```

Reply:
Neat!  Updated.